services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/api/frontend/routes/_service.py

```diff
@@ -25,9 +25,9 @@ def _render_remove_from_tracking(node_id):
 
         async def remove_from_tracking():
             confirm_dialog.close()
-            await httpx.AsyncClient(timeout=10).get(
-                f"http://localhost:{DEFAULT_FASTAPI_PORT}/service/{node_id}/tracker:remove"
-            )
+
+            url = f"http://localhost:{DEFAULT_FASTAPI_PORT}{get_settings().DYNAMIC_SCHEDULER_UI_MOUNT_PATH}service/{node_id}/tracker:remove"
```

Comment:
I would use a URL object/tool to compose this and avoid mistakes

Reply:
Tried it, but does not bring anything at all (nor safety, nor does it help me deal with my edge case). I decided to drop it. Just makes the code more unreadable.
Tried both `yarl.URL` and `pydantic.AnyUrl`